rdkit/Chem/UnitTestInchi.py

```diff
@@ -175,7 +175,9 @@ def test1InchiReadPubChem(self):
           continue
         x = MolToInchi(m)
         y = None
+        RDLogger.DisableLog('rdApp.error')
```

Comment:
Again, I don't think this needs to be done quite so fine-grained. It can be moved up and outside the loop